package.json

```diff
@@ -6,6 +6,15 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
+  "exports": {
+    ".": "./index.js",
+    "./js": "./index.js",
+    "./js.js": "./index.js",
+    "./ts": "./ts.js",
+    "./ts.js": "./ts.js",
+    "./jest": "./jest.js",
+    "./jest.js": "./jest.js"
```

Comment:
Why both with and without extension?

Reply:
We have `'import/extensions': ['error', 'always'],`, we'd have to override it. I just enabled using the config with extensions, which doesn't hurt anybody

Reply:
And that applies even when importing subpath exports? That's dumb.

How about changing the `import/extensions` rule to ignore it for this package?

```js
'import/extensions': [
    'error',
    'always',
    {
        checkTypeImports: true,
        pathGroupOverrides: [{
            pattern: '@apify/eslint-config/*',
            action: 'ignore',
        }],
    },
]
```

Reply:
sry Franta, the code you provided is not released yet.

https://github.com/import-js/eslint-plugin-import/commit/fa36d4983c03f1e3a63c3cd555ea2fb34adc0a86